src/components/tvshow.jsx

```diff
@@ -40,6 +40,7 @@ export default function TvShow() {
 
     return (
         <div>
+            <ResponsiveAppBar/>
```

Comment:
Instead of having the navigation bar in the trending shows page, let's at it outside so it will be rendered in all pages without us having to add it to every page